source/general.rst

```diff
@@ -0,0 +1,62 @@
+.. _general:
+
+##########################
+General Project Guidelines
+##########################
+
+Code of Conduct
+---------------
+
+All members participating in the yt project community, whether that be on
+github, mailing lists, workshops, or in any other capacity are expected to
+abide by the `code of conduct
+<https://yt-project.org/doc/developing/developing.html#yt-community-code-of-conduct>`_.
+
+If any person feels a member of the yt project community has violated the code
+of conduct, a report can be made to a project member or sent to
+confidential@yt-project.org. Reports of CoC violations will be treated with the
+strictest confidence.
+
+Where the Governance Applies
+----------------------------
+
+This governance document applies to all repositories hosted within the yt
+project organization. If a package proposes an alteration to this governance
+model, it can be further refined in that repository, but must be implemented
+through the following procedure:
+
+* A pull request is made to that repository with a proposed alternative 
+  governance model
+* The pull request is announced in public channels available to the community and 
+  must be discussed at the next steering committee meeting.
+* The alternative model is accepted by more than half of the active maintainers 
```

Comment:
Not in our documentation yet. My instinct would be to define it as somebody who has reviewed a PR within x amount of time, but maybe we want to leave it ambiguous? I could add a definition somewhere if we want to define it.

Reply:
Can we also proactively define a minimum cutoff time for participation?  Otherwise we would be in the situation that people that maybe are a bit tuned out might miss it.